### PR TITLE
Add new results

### PR DESCRIPTION
**Pull request by scottgigante-immunai**

### CODE REVIEW

**Comment by scottgigante-immunai** [comment]
cc @sumneuron

**Comment by dburkhardt** [comment]
This deploy isn't working

image

**Comment by dsm-72** [comment]
@scottgigante-immunai all the CPU%s are like in the 1000s

**Comment by scottgigante** [comment]
@dburkhardt that won't matter if we ditch Hugo, right?

@dsm-72 that just means it was using on average 13 cores

**Comment by rcannood** [comment]
The deploy wasn't working because the data/results/*.json files ned to have unique names. I manually changed the name in this PR, but I assume @scottgigante-immunai will have to update this in an upstream script.

In addition, I also split up the results menu into two separate submenu's, borrowing from the Online course template ([Demo](https://hugo-online-course.netlify.app/), [Source](https://github.com/wowchemy/starter-hugo-online-course/tree/a864e0b544a6758dea58df233822743a4919c36e)).

File: data/results/multimodal_data_integration/citeseq_cmbc.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "CITE-seq Cord Blood Mononuclear Cells",
+    "name": "CITE-seq Cord Blood Mononuclear Cells (duplicate)",
```
Comment (scottgigante-immunai) [inline data/results/multimodal_data_integration/citeseq_cmbc.json:2]: So this is not ideal -- in this case we have two tasks using the same dataset, they should really have the same name.

Comment (rcannood) [reply data/results/multimodal_data_integration/citeseq_cmbc.json:2]: I assumed that the JSON files are auto generated so I only made the minimal changes required to get it to work, not the way it should be before this get merged. Sorry for not making this clear from my comment.

I think the main issue is that there is a citeseq_cmbc.json as well as a citeseq_cbmc.json for the same task. Is this a typo?

Comment (scottgigante-immunai) [reply data/results/multimodal_data_integration/citeseq_cmbc.json:2]: Oh now I see it! Yes one of them is a typo.

File: config/_default/menus.yaml [hunk 2]
```diff
@@ -10,6 +10,15 @@ main:
   - name: Results
     url: /results
     weight: 20
+    identifier: results
+  - name: Label projection
+    url: /results/label_projection
+    parent: results
+    weight: 10
+  - name: Multimodal data integration
```
[the remark this answers is not in the document]
Comment (scottgigante-immunai) [reply config/_default/menus.yaml:18]: Gotcha. Let's move this to a separate PR?

[the remark this answers is not in the document]
Comment (rcannood) [reply config/_default/menus.yaml:18]: Sure, that makes sense!

**Comment by scottgigante-immunai** [comment]
Thanks. I'll leave it to you and Sumner to set up that as a separate PR

**Comment by rcannood** [comment]
Just noticed. @scottgigante There is also `dimensionality_reduction` and `dimensional_reduction` in the results folder. Perhaps these need to be trimmed as well?

**Comment by scottgigante-immunai** [comment]
Amazing :)

**Comment by scottgigante-immunai** [comment]
@dburkhardt I don't think I have merge access. Mind doing the honours?

**Pull request merged**